NIRoberto/todo-app
# ft[add wrestle name route ]

**What does this PR do?**

- Add endpoint where you should be able to create wrestle name, delete wrestle name, update and get wrestle name



**Description of Task to be completed?**

- You need to have an account to my app if not create it 



**How should this be manually tested?**

- clone this project into your system using `git clone https://github.com/NIRoberto/todo-app.git`
- Install packages using `npm install ` command.
- Run `npm run dev ` command to start the server 
- Open Postman and send the post method for all login and sign up using this URL `localhost:2000/api/v1/todo/wrestles/ ` for get all wrestle name `localhost:2000/api/v1/todo/wrestles/:id ` for get one localhost:2000/api/v1/todo/wrestles/update/:id ` for update one localhost:2000/api/v1/todo/wrestles/delete/:id ` for delete one
- In  body pass JSON with a valid wrestle name  name
- Make sure you are sending the request with  specified method example if it is delete send `delete` request
- Send request

**Any background context you want to provide?**

**What are the relevant Trello board stories?**
- https://trello.com/c/dOxwYyvS
- https://trello.com/c/zE9RPFyn
- https://trello.com/c/Iz6IQnyn
- https://trello.com/c/Aq3BQ0Ur
- https://trello.com/c/au2w7j49

**Screenshots (if appropriate)**
